pkg/clustermesh/clustermesh.go

```diff
@@ -356,64 +246,15 @@ func (cm *ClusterMesh) NumReadyClusters() int {
 	return nready
 }
 
-func (cm *ClusterMesh) canConnect(name string, config *cmtypes.CiliumClusterConfig) error {
+func (cm *GenericClusterMesh) Iterator(fn func(RemoteCluster) error) error {
```

Comment:
Nit: For me, `Iterator` is the name of the function that returns an iterator object. I'd name this kind of function with `ForEachRemoteCluster` or something. Just a matter of the preference though 😓

Reply:
Yeah, sounds good to me. I've renamed this method to `ForEachRemoteCluster`.